integrations/mcp-server/Dockerfile

```
@@ -19,10 +24,17 @@ RUN pnpm --filter @tsparticles/mcp-server exec tsc
 RUN pnpm --filter @tsparticles/mcp-server deploy --prod /tmp/app
 
 FROM node:22-alpine
+# tini acts as a proper PID 1: reaps zombie processes and forwards
+# signals (SIGTERM/SIGINT) to the Node process so graceful shutdown
+# actually runs instead of the container being killed outright.
+RUN apk add --no-cache tini
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)